src/NUnitFramework/framework/Internal/Extensions/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System.Collections;
+using System.Collections.Specialized;
+using System.Linq;
+
+namespace NUnit.Framework.Internal.Extensions
+{
+    internal static class IEnumerableExtensions
+    {
+        public static bool IsSortable(this IEnumerable collection)
+        {
+            if (collection is null)
+                return false;
+
+            if (collection is StringCollection)
```

Comment:
I think this is the only non-generic built in collection that is strongly typed?